amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -36,16 +36,21 @@ CoriolisForcing::CoriolisForcing(const CFDSim& sim)
     m_sinphi = std::sin(m_latitude);
     m_cosphi = std::cos(m_latitude);
 
-    // Read the rotational time period (in seconds)
-    amrex::Real rot_time_period = 86400.0;
+    // Read the rotational time period (in seconds) -- This is 23hrs and 56
+    // minutes and 4.091 seconds
+    amrex::Real rot_time_period = 86164.091;
     pp.query("rotational_time_period", rot_time_period);
-    m_coriolis_factor = 2.0 * utils::two_pi() / rot_time_period;
+
+    m_omega = 2.0 * utils::two_pi() / rot_time_period;
```

Comment:
I don't agree here.  Omega should be 2 pi radians divided by the rotation period.  I don't understand the premultiplying by 2.

Reply:
good catch, Matt. I agree that the calculation of omega should just be 2pi / rot_time_period. The new code can add clarity by showing m_omega directly but it shouldn't be numerically different.

Reply:
Haha, that's a good catch indeed. I hostly thought this was `utils::pi()` and I completely overlooked the two_ in front of it